src/Repository/RepoRepository.php

```diff
@@ -27,11 +29,28 @@ public function findOfType($provider)
         return $qb->getQuery()->getResult();
     }
 
+    /**
+     * @throws Exception
+     */
+    public function deleteOfType($provider, $orgName)
+    {
+        $conn = $this->getEntityManager()->getConnection();
+
+        $sql = '
+            DELETE FROM repo AS r USING org AS o WHERE r.org_id = o.id AND o.name = :orgName AND r.provider = :provider
+            ';
+        $stmt = $conn->prepare($sql);
+
+        $stmt->executeQuery(['provider' => $provider,'orgName' => $orgName]);
```

Comment:
```suggestion
        $conn = $this->getEntityManager()->getConnection();

        $sql = '
            DELETE FROM repo AS r USING org AS o WHERE r.org_id = o.id AND o.name = :orgName AND r.provider = :provider
            ';
        $conn->executeStatement($sql, ['provider' => $provider,'orgName' => $orgName]);
```

We can use dbal connection methods.

More DBAL connection api here:
https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/data-retrieval-and-manipulation.html